x-pack/plugins/apm/server/routes/settings/apm_indices/save_apm_indices.ts

```diff
@@ -19,12 +19,9 @@ export function saveApmIndices(
 ) {
   return withApmSpan('save_apm_indices', () =>
     savedObjectsClient.create(
-      APM_INDICES_SAVED_OBJECT_TYPE,
-      removeEmpty(apmIndices),
-      {
-        id: APM_INDICES_SAVED_OBJECT_ID,
-        overwrite: true,
-      }
+      APM_INDEX_SETTINGS_SAVED_OBJECT_TYPE,
+      { ...removeEmpty(apmIndices), isSpaceAware: true },
```

Comment:
I think it is a little unfortunate that we mix the index settings, and meta data for the saved object (`isSpaceAware`).

Can we do something like this:

```suggestion
      { indexSettings: removeEmpty(apmIndices), isSpaceAware: true },
```

Reply:
done it here https://github.com/elastic/kibana/pull/126176/commits/873e1d4a3ea0d289d967c5dfe6c2bd72d5848434